src/ripple/core/impl/Config.cpp

```diff
@@ -162,7 +162,12 @@ void Config::setup (std::string const& strConf, bool bQuiet)
     // config directory and that with "db" as the data
     // directory.
 
-    QUIET = bQuiet;
+    QUIET = bQuiet || bSilent;
```

Comment:
Since you've moved the initialization `QUIET`, `SILENT` and `RUN_STANDALONE`, if we never _modify_ those variables outside of `Config`, then consider making them `private` and adding three member functions:

    bool quiet() const { return QUIET; }
    bool silent() const { return SILENT; }
    bool standalone() const { return STANDALONE; }